docs/generated/workloads.md

```diff
@@ -3864,6 +3864,26 @@ The queries in this workload exercise group stage that uses an enum like field f
 timeseries, aggregate, group 
 
 
+## [TimeseriesExtendedRange](https://www.github.com/mongodb/genny/blob/master/src/workloads/query/TimeseriesExtendedRange.yml)
+### Owner 
+Query Integration 
+
+
+### Support Channel
+[#query-integration](https://mongodb.enterprise.slack.com/archives/C04PDS7GAFM)
```

Comment:
I would say that we shouldn't be linking to slack channels on a public readme but looks like we are already doing that elsewhere, and removing all of those links is definitely outside of scope for this ticket.

Reply:
Yea, this was auto-generated. I'm guessing the slack is private so the link just goes nowhere for public viewers